php/libraries/FilesUploadHandler.class.inc

```diff
@@ -0,0 +1,129 @@
+<?php
+/**
+ * An interface containing methods for classes used to handle file
+ * uploads from a client to a server.
+ *
+ * PHP Version 7
+ *
+ * @category Main
+ * @package  Main
+ * @author   Nicolas Brossard <nicolasbrossard.mni@gmail.com>
+ * @license  http://www.gnu.org/licenses/gpl-3.0.txt GPLv3
+ * @link     https://www.github.com/aces/Loris/
+ */
+
+/**
+ * Handles file uploads from a client to a server. The methods listed in this
+ * interface were created in order to encapsulate the usual things that happen
+ * when a set of files is uploaded to a server, namely:
+ *
+ * 1. Validation of all uploaded files
+ * 2. Transfer of all files to a designated (permanent) location on the
+ *    server.
```

Comment:
Why do these need to be all files and not (potentially) some files?

Reply:
I would effectively remove the "all"